internal/store/ingress.go

```diff
@@ -202,6 +202,28 @@ func ingressMetricFamilies(allowAnnotationsList, allowLabelsList []string) []gen
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_ingress_status",
```

Comment:
```suggestion
			"kube_ingress_status",
```
I'm a bit torn on this name.
Maybe:
kube_ingress_status_loadbalancer